src/CrateTokenV1.sol

```diff
@@ -1,41 +1,31 @@
 //SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.24;
 
-import "lib/openzeppelin-contracts-upgradeable/contracts/token/ERC20/ERC20Upgradeable.sol";
+import "@openzeppelin-upgradeable/contracts/token/ERC20/ERC20Upgradeable.sol";
 import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
-import "../lib/multicaller/src/LibMulticaller.sol";
-import {IUniswapV2Router02} from "../src/interfaces/IUniswapV2RouterV2.sol";
+import {IUniswapV2Router02} from "src/interfaces/IUniswapV2RouterV2.sol";
+import {ICrateV1} from "src/interfaces/ICrateV1.sol";
 
 // The total supply is 117,000 tokens
 // Once the bonding curve has sold out 80,000 tokens, the other 37,000 are put in Uniswap with the total ETH in the contract.
 // The LP tokens are then burned, so no one can pull the liquidity.
+contract CrateTokenV1 is ERC20Upgradeable, ReentrancyGuard, ICrateV1 {
+    uint256 private constant MAX_SUPPLY = 117_000e18;
+    uint256 private constant MAX_CURVE_SUPPLY = 80_000e18;
+    uint256 private constant CROWDFUND_THRESHOLD = 60_000e18;
```

Comment:
I'm assuming `CROWDFUND_THRESHOLD` is only temporarily set to `60_000e18`, this pattern will eventually switch to a USDC goal right?

This is fine for now just confirming

Reply:
Yes, exactly